create-wic.sh

```diff
@@ -83,17 +83,22 @@ source ${topdir}/scripts/common.sh
 
 validate_section "Build" ${BUILD} "${topdir}/builds.toml"
 
-if [ ! -f ${topdir}/build/${BUILD}/tisdk-debian-${BUILD}-boot.tar.xz ]; then
+
+distro_codename=($(read_build_config ${BUILD} distro_codename))
+bsp_version=($(read_bsp_config ${distro_codename} bsp_version))
+BUILD_FULL="${BUILD}-${bsp_version}"
```

Comment:
Might be more clear what file is being used if you drop `BUILD_FULL` and just use `${BUILD}-${bsp_version}` directly below.

Reply:
Updated the PR.